### ISSUE TITLE
Breast abscess

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10093]
**Preferred term label:**
Breast abscess

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0151463

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Breast abscess

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as 
HP:0020225
